### DISCUSSION TITLE
Brush event data of echarts

### DISCUSSION THREAD

**Discussion by mswastik** [zauberzeug/nicegui#2308] [Ideas / Feature Requests]
There is already `on_point_click` parameter in `ui.echart` function to make point click data from echart available in python. Is it also possible to make brush data available in python? Details of brush selection on echarts can be found  at [https://echarts.apache.org/en/option.html#brush](https://echarts.apache.org/en/option.html#brush).

**Comment by falkoschindler** [comment]
Hi @mswastik!

Currently, `ui.echart` only supports point click events. Adding brush or even more events would be a great new feature.
Or maybe we find a way to generically support _all_ events, but I'm not sure how.

Ideas, pull requests and other contributions are greatly appreciated. 🙂
